src/python/pants/bin/local_pants_runner.py

```diff
@@ -82,12 +81,8 @@ class LocalPantsRunner(object):
   """Handles a single pants invocation running in the process-local context."""
 
   @staticmethod
-  def parse_options(args, env, setup_logging=False, options_bootstrapper=None):
+  def parse_options(args, env, options_bootstrapper=None):
     options_bootstrapper = options_bootstrapper or OptionsBootstrapper.create(args=args, env=env)
-    bootstrap_options = options_bootstrapper.get_bootstrap_options().for_global_scope()
-    if setup_logging:
```

Comment:
So, I believe that when `LocalPantsRunner` is running alone (without pantsd) it will still need to set up logging. So I think this conditional should still be here?

Reply:
I removed the conditional because fwics, in the alone case, we are setting rust logging here - https://github.com/pantsbuild/pants/blob/master/src/python/pants/bin/pants_runner.py#L52
- lmk if thats not the case

Reply:
Makes sense to me! ...and assuming that logging is still working for `--no-enable-pantsd`, you're good either way =)